src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { preparePeople } from '../../helpers';
+import { Person } from '../../types';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const { slug = '' } = useParams();
+
+  const fetchPeople = async () => {
+    try {
+      let data = await getPeople();
+
+      data = preparePeople(data);
+      setPeople(data);
+      setIsLoading(false);
+    } catch {
+      setHasError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const isPeopleExist = useMemo(() => {
+    return people.length === 0 && !isLoading;
+  }, [people]);
```

Comment:
in this case, useMemo is not good, you don't perform complex operations with the object, such as filtering or mapping, comparing values is a simple operation, useMemo is overkill here